locust/runners.py

```diff
@@ -293,6 +309,7 @@ def __init__(self, id, state=STATE_INIT, heartbeat_liveness=3):
         self.state = state
         self.user_count = 0
         self.heartbeat = heartbeat_liveness
+        self.cpu_threshold_exceeded = False
```

Comment:
I think it would be better to store the CPU usage value on the `SlaveNode` and track on the runner if we've emitted a warning. Then it'll be very easy to display CPU usage on the slave nodes tab in the web UI.